Tests/CPP_Bindings/Source/Reader.cpp

```diff
@@ -149,4 +149,10 @@ namespace Lib3MF
 		auto reader = model->QueryReader("3mf");
 		ASSERT_SPECIFIC_THROW(reader->ReadFromFile(sTestFilesPath + "/Reader/" + "GEN-M-ADA-ITEM-TRANSFORM-0.3mf"), ELib3MFException);
 	}
+
+
+	TEST_F(Reader, IntegrationTestError) {
```

Comment:
good idea to add a test there.

Please explain what the content of the file is and why it should fail.
Also please change the name to something meaningfull.